regression/cbmc-primitives/CMakeLists.txt

```diff
@@ -5,11 +5,11 @@ if(Z3_EXISTS)
         "$<TARGET_FILE:cbmc>"
     )
 
-    # If `-I` (include flag) is passed, test.pl will run only the tests matching the label following it.
+    # If `-X` (include flag) is passed, test.pl will exclude the tests matching the label following it.
     add_test_pl_profile(
             "cbmc-primitives-new-smt-backend"
             "$<TARGET_FILE:cbmc> --incremental-smt2-solver 'z3 --smt2 -in'"
-            "-I;new-smt-backend;-s;new-smt-backend"
+            "-X;no-new-smt;-s;new-smt-backend"
```

Comment:
This looks off, but I don't recall the exact syntax here.

Reply:
I'm not convinced there is anything wrong here? We want to flip to selectively _excluding_ rather than selectively _including_ tests.

Reply:
I clearly was not really clear. The change from `-I` to `-X` is correct, but there remains the `-s;new-smt-backend` which looks off to me. That said, I don't recall the syntax (and should check).

Reply:
I guess that's ok for it's an arbitrary string to disambiguate log files:
```
$ regression/test.pl --help
[...]
  -s <suffix>  append <suffix> to all output and log files. Enables concurrent
             testing of the same desc file with different commands or options,
             as runs with different suffixes will operate independently and keep
             independent logs.
[...]
```

Reply:
As per the previous comment the `-s` disambiguates log files. So updating it to contain "no" for example would seem incorrect/misleading. It would match the tag but appear in the file name of output files where the "no" tag is not present. @TGWDB Ok if I consider this comment to be resolved?